snippets/xml_snippets.json

```diff
@@ -415,10 +415,17 @@
         ],
         "description": "Add link tag to include stylesheet"
     },
+    "Self-closing field tag": {
+        "prefix": "oofield",
+        "body": [
+            "<field name=\"${1}\"${20}/>${50}"
+        ],
+        "description": "Add a self-closing field tag"
+    },
     "Field tag": {
         "prefix": "oofield",
         "body": [
-            "<field name=\"${1}\"></field>"
+            "<field name=\"${1}\"${20}>${30}</field>${50}"
```

Comment:
`<field name=\"${1}\">${0}</field>`

Reply:
Same here.

Reply:
This allow developer to add field name and it's value. e.g. Demo data record